contrib/storage-hbase/src/main/java/org/apache/drill/exec/store/hbase/HBaseRecordReader.java

```diff
@@ -186,6 +192,10 @@ public void setup(OperatorContext context, OutputMutator output) throws Executio
           }
         }
       }
+
+      for (String familyName : completeFamilies) {
+        getOrCreateFamilyVector(familyName, false);
+      }
```

Comment:
Does this create just the map, or also the vectors within the map? Maybe a comment to explain the goals?

Reply:
It creates only the map vector